geoips/interfaces/yaml_based/sectors.py

```diff
@@ -5,6 +5,8 @@
 
 from geoips.interfaces.base import BaseYamlPlugin, BaseYamlInterface
 
+# from geoips.pydantic.sectors import SectorPluginModel
+
```

Comment:
What are these commented out lines for?  If they are required, probably note why they are here.

Reply:
These are for enabling the pydantic models once we have finished implementing dynamic sectors. I had enabled this but it only works for static sectors for now. Adding a comment.

Reply:
Thank you